fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -102,17 +104,17 @@ const RollCallOpened = () => {
   const addAttendeePopToken = useCallback(
     (popToken: string) => {
       // if the token is already part of attendeePopTokens, do not trigger a state update
-      if (attendeePopTokens.has(popToken)) {
+      // and return false indicating the pop token as not added since it's a duplicate
+      if (attendeePopTokens.includes(popToken)) {
         return false;
       }
 
-      updateAttendeePopTokens(
-        // use new Set() to trigger a state change. .add() would not still be the same object
-        (prevAttendeePopTokens) => new Set<string>([...prevAttendeePopTokens, popToken]),
-      );
+      navigation.setParams({
```

Comment:
Is `attendeePopTokens` the only information in the navigation params? 
If not, we might need to combine with other existing params.
Actually, I'd consider doing that anyways, to future-proof the code.

Reply:
It already does it for us as react's setState for class components ;) https://reactnavigation.org/docs/navigation-prop/#setparams

Reply:
oh, even better ! I guess I learned one more thing today 😅